src/App.css

```diff
@@ -0,0 +1,42 @@
+#root {
```

Comment:
Remove the App.css file
Use the Mui styled components. It should be like (just example):
```
const BoxStyled = styled(Box)({
  ...
})
```

Reply:
done